src/classes/Frames.ts

```diff
@@ -0,0 +1,49 @@
+import { ErrorType } from "../types";
+
+export const getTFRFrames = async ({ sku }): Promise<ErrorType | void> => {
+  window.theFittingRoom?.getUserProfile().then(userProfile => {
+    console.log("userProfile: ", userProfile?.vto);
+    console.log("sku: ", sku);
+
+    const getBrandId = () => {
+      const brandStyleId = sku?.split('-')?.[0];
+      return String(brandStyleId || "");
+    }
+
+    const getUserFrames = () => {
```

Comment:
Similar to the top level function name, this is ambiguous. We should specify that its getting the vto frames